web/packages/api/src/xcmBuilder.ts

```diff
@@ -332,6 +368,27 @@ export function buildParachainERC20ReceivedXcmOnAssetHub(
     })
 }
 
+function buildAssetHubXcmFromParachainKusama(beneficiary: string, topic: string) {
+    return [
```

Comment:
Would it be better to move these functions to a separate file specific to kusama, e.g., kusamaXcmBuilder.ts?

Reply:
Definitely, addressed in https://github.com/Snowfork/snowbridge/pull/1463/commits/2fed1d60dffa676c1263bb2d6de4df985e8f1576.